docs/docs-content/deployment-modes/agent-mode/install-agent-host.md

```diff
@@ -154,6 +154,86 @@ Palette. You will then create a cluster profile and use the registered host to d
      semodule --install rsync_dac_override.pp
      ```
 
+  7. (Optional) If you are using Cilium, issue the following commands to configure the appropriate firewalld rules.
```

Comment:
```suggestion
  7. (Optional) If you are using Firewalld and Cilium, issue the following commands to configure the appropriate firewalld rules.
```

Reply:
Also, these commands are meant to be run as part of a script since they take an argument, `$1`. If you put these commands in a file let's call it `firewalld-k8s-cilium.sh` then you would need to execute like so: `./firewalld-k8s-cilium.sh public`. Where `public` is the name of the active firewalld zone.